src/app/utilities/websocket.js

```diff
@@ -62,7 +62,7 @@ class Socket {
 
         if (this.buffer.size >= 50) {
             console.warn(`Websocket buffer has reached it's limit, so message will not be sent to server. Message: \n`, message);
-            log.add(eventTypes.socketBufferFull); // This has to be excluded from being sent to the server or else we'll have an infinite loop
+            //log.add(eventTypes.socketBufferFull); // This has to be excluded from being sent to the server or else we'll have an infinite loop
```

Comment:
would normally say we should remove commented out code, but this feels more like a comment that might be better left in?